spaceship/lib/spaceship/portal/certificate.rb

```diff
@@ -211,6 +211,23 @@ def create_certificate_signing_request
           return [csr, key]
         end
 
+        def create_apple_pay_certificate_signing_request
+          OpenSSL::PKey::EC.send(:alias_method, :private?, :private_key?)
+
+          ec_domain_key = OpenSSL::PKey::EC.new('prime256v1')
+          ec_public = OpenSSL::PKey::EC.new('prime256v1')
+          ec_domain_key.generate_key
+          ec_public.public_key = ec_domain_key.public_key
+
+          csr = OpenSSL::X509::Request.new
+          csr.subject = OpenSSL::X509::Name.new([
+                                                  ['CN', 'PEM', OpenSSL::ASN1::UTF8STRING]
```

Comment:
Is "Common Name" really `PEM` here? I think this is properly not surfaced anywhere, but this should probably be `spaceship` or something similar here, right?

Reply:
We have a similar logic for `create_certificate_signing_request` method as well.

Reply:
Yes, that is used by `PEM` and most probably a historical artifact. You are creating new code, so it should make sense _now_.

Reply:
Make sense. I have changed it for the legacy code as well.